cmd/dusk/monitor.go

```diff
@@ -4,13 +4,15 @@ import (
 	"strings"
 	"time"
 
+	"github.com/sirupsen/logrus"
```

Comment:
Seems like logrus is imported twice now. Let's stick to just one, to keep the code clean :)

Reply:
done